frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -320,6 +323,59 @@ export class EnrolledItemCard extends React.Component<
     )
   }
 
+  renderUnverifiedUnenrollmentModal(enrollment: RunEnrollment) {
+    const { runUnenrollmentModalVisibility } = this.state
+    const now = moment()
+    const endDate = enrollment.run.enrollment_end
+      ? parseDateString(enrollment.run.enrollment_end)
+      : null
+    const formattedEndDate = endDate ? formatPrettyDateTimeAmPmTz(endDate) : ""
+    return (
+      <Modal
+        id={`run-unenrollment-${enrollment.run.id}-modal`}
+        className="text-center"
+        isOpen={runUnenrollmentModalVisibility}
+        toggle={() => this.toggleRunUnenrollmentModalVisibility()}
+        role="dialog"
+        aria-labelledby={`run-unenrollment-${enrollment.run.id}-modal-header`}
+        aria-describedby={`run-unenrollment-${enrollment.run.id}-modal-body`}
+      >
+        <ModalHeader
+          id={`run-unenrollment-${enrollment.run.id}-modal-header`}
+          toggle={() => this.toggleRunUnenrollmentModalVisibility()}
+        >
+          Unenroll From {enrollment.run.title}
+        </ModalHeader>
+        <ModalBody id={`run-unenrollment-${enrollment.run.id}-modal-body`}>
+          <p>
+            Are you sure you wish to unenroll from {enrollment.run.title}?
+            {endDate
+              ? now.isAfter(endDate)
+                ? "You won't be able to re-enroll"
+                : `You won't be able to re-enroll after ${formattedEndDate}`
+              : null}
```

Comment:
Needs a space between the `Are you sure` message and the date-specific messaging, and needs a period after the date-specific lines.